.travis.yml

```diff
@@ -11,6 +11,9 @@ language: python
 python:
   - "2.7"
 
+cache:
+  - pip
```

Comment:
As discussed else where, if there is a way to cache the third-party binaries (ex: graphviz) built for each time, that would reduce the build time even further.